app/waf/src/models/application.model.ts

```diff
@@ -33,17 +33,17 @@ export class Application extends Entity {
   @property({
     type: 'date',
   })
-  created_at?: string;
+  createdAt?: string;
```

Comment:
Can we hold a while and carefully decide with naming format is better for us?

Reply:
Not sure about it. 
Before knowing the best solution, it's better to keep all keys with same naming format. 
`{"wafpolicyId": "1234", "createAt": "2019"}` gives consistant feeling than  `{"wafpolicyId": "1234", "create_at": "2019"} `

Reply:
Let us merge the code now, and talk about the naming later when we refining the API.